docs/releases/1.11.txt

```diff
@@ -101,6 +101,11 @@ Minor features
 * :func:`~django.contrib.auth.update_session_auth_hash` now rotates the session
   key to allow a password change to invalidate stolen session cookies.
 
+* The new attribute ``success_url_allowed_hosts`` for
+  :class:`~django.contrib.auth.views.LoginView` and
+  :class:`~django.contrib.auth.views.LogoutView` allows specifying a set of
+  hosts that are safe for redirect after login and log out.
```

Comment:
> is login/logout one word or two? (or are they really different?)

Seems like all four spellings are used to some degree. I believe I initially copied "login" and "log out"  from some other documentation, assuming they were the preferred spelling. A quick grep and it looks like "login" and "logout" are much more common, so I'll stick with that here unless you have a particular preference.

```
$ find docs/ -name \*.txt -exec grep -nH '\blogin\b' {} \; | wc -l
193
$ find docs/ -name \*.txt -exec grep -nH '\blog in\b' {} \; | wc -l
19
$ find docs/ -name \*.txt -exec grep -nH '\blogout\b' {} \; | wc -l
52
$ find docs/ -name \*.txt -exec grep -nH '\blog out\b' {} \; | wc -l
8
```